bpfman-operator/apis/v1alpha1/uprobeProgram_types.go

```diff
@@ -77,7 +77,7 @@ type UprobeProgramSpec struct {
 	Pid int32 `json:"pid"`
 
 	// Containers identifes the set of containers in which to attach the uprobe.
-	// If Containers are not specified, the uprobe will be attached in the
+	// If Containers is not specified, the uprobe will be attached in the
 	// bpfman-agent container.  The ContainerSelector is very flexible and even
```

Comment:
Andrew:
I think being able to easily specify "all containers in a cluster" Is a footgun we shouldn't expose

Andre:
I started out making it more opinionated to avoid the footguns.  However, I found this approach less user friendly.  There are times when you might want to include (or not) each of the parameters.  In the end I came to the opinion that this is pretty low level functionality, and you should know what you're doing to use it.  So, I'd rather not make it harder to use by arbitrarily limiting how you can choose the containers.

Reply:
That's a fair enough argument for me! Let's just make sure to have documentation/ comments that say as much.